onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -83,6 +83,7 @@
     os.environ["OMP_NUM_THREADS"] = str(cpu_count)
 
 import torch
+import tensorflow as tf
```

Comment:
import tensorflow only when needed. For example, user only need benchmark pytorch, there is no need to install tensorflow.